packages/api/src/utils/helpers.ts

```diff
@@ -407,3 +408,16 @@ export const isRelativeUrl = (url: string): boolean => {
 export const getAbsoluteUrl = (url: string, baseUrl: string): string => {
   return new URL(url, baseUrl).href
 }
+
+export const setRecentlySavedItemInRedis = async (
+  userId: string,
+  url: string
+) => {
+  // save the url in redis for 8 hours so rss-feeder won't try to re-save it
+  const redisKey = `recent-saved-item:${userId}:${url}`
+  const ttlInSeconds = 60 * 60 * 8
+  return redisClient.set(redisKey, 1, {
```

Comment:
one question: does this throw an exception? I think if this set fails (or if the get fails), we still want to save the item and not fail the job,.

Reply:
yeah, this will throw an exception. let me change the part to catch the error and log